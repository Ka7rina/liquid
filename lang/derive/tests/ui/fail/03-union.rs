use liquid_lang as liquid;

use liquid::InOut;
#[derive(InOut)]
#[repr(C)]
union MyUnion {
    f1: u32,
    f2: f32,
}

fn main() {}
