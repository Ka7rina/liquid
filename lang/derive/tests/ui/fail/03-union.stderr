error: unions are not supported to derive from `InOut` or `State`
 --> $DIR/03-union.rs:6:1
  |
6 | union MyUnion {
  | ^^^^^
